app/controllers/games_controller.rb

```diff
@@ -19,7 +19,17 @@ def piece_update
 		@game = Game.find(params[:game_id])
 		@piece = Piece.find(params[:id])
 		@piece.update_attributes(:x_axis => params[:x_axis], :y_axis => params[:y_axis])
-		redirect_to game_path(@game)
+
+		if King.checked?(King.current_position)
```

Comment:
I think you don't want to use a static method `checked?`.  Instead I think you want to call it on the actual item.

Something like:

```
@game.pieces.kings
```

Or something.  I think we'd need to check both colors.